src/ios/CDVNavitiaSDKUI.swift

```diff
@@ -31,6 +31,7 @@ import NavitiaSDKUI
         let destinationColor = toUIColor(hexColor: config["destinationColor"] as? String) ?? UIColor(red:0.69, green:0.01, blue:0.33, alpha:1.0)
         let multiNetwork = config["multiNetwork"] as? Bool ?? false
         let formJourney = config["formJourney"] as? Bool ?? false
+        let isEarlierLaterFeatureEnable = config["isEarlierLaterFeatureEnable"] as? Bool ?? false 
```

Comment:
Can we have a better name for this feature parameter ? And anyway, `isEarlierLaterFeatureEnable` should be `isEarlierLaterFeatureEnable**d**` or `enableEarlierLaterFeature`

Reply:
done